### ISSUE TITLE
Team Meeting Cadence

### ISSUE THREAD

**Issue by hindeb** [bahrain-bp/mun-aquacrop#16]

**Comment by chebibj** [comment]
When is the Cadence?  
Can you provide the details here for the weekly meetings?

**Comment by izZERO** [comment]
2 times a week , On Monday & Wednesday.

**Comment by izZERO** [comment]
**Meeting Summary**

Date: 11/14/2024
Time: 7:00 pm
Attendees: Hasan , Ali Annan, Ali Fardan

**Agenda:**
-Progress Updates
-Frontend Template Discussion and Decision

**Discussion:**
1. Progress Updates
We discussed individual task progress, sharing any blockers and next steps. We each provided an overview of where we currently stand in our respective tasks.

2. Frontend Template Discussion and Decision
We reviewed multiple front-end templates for the mobile app that are compatible with React Native, then selected one of them as the preferred front-end template for the app.

**Next Steps:**
Implement the chosen template in upcoming development tasks.
Continue with individual tasks, integrating the new template as necessary.
